docs/standard/garbagecollection/fundamentals.md

```diff
@@ -151,7 +151,7 @@ Before a garbage collection starts, all managed threads are suspended except for
 
 The following illustration shows a thread that triggers a garbage collection and causes the other threads to be suspended.
 
-! [When a thread triggers a garbage collection](../images/IC393001.png)
+! [When a thread triggers a garbage collection](../media/IC393001.png)
```

Comment:
the file needs to be inside media and then inside a folder with the same name as the article, so it would be media/fundamentals.